File: CD/ex.c
#include <stdio.h>

void switch_case_example(int value) {
    switch (value) {
        case 1:
            printf("Case 1\n");
            break;
        case 2:
            printf("Case 2\n");
            break;
        case 3:
            printf("Case 3\n");
            break;
        default:
            printf("Default case\n");
    }
}

int main() {
    int value = 2;
    switch_case_example(value);
    return 0;
}

File: CD/ex.java
public class ex {

    public static void switch_case_example(int value) {
        switch (value) {
            case 1:
                System.out.println("Case 1");
                break;
            case 2:
                System.out.println("Case 2");
                break;
            case 3:
                System.out.println("Case 3");
                break;
            default:
                System.out.println("Default case");
        }
    }

    public static void main(String[] args) {
        int value = 1;
        switch_case_example(value);
    }
}
